consensus/poet/core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -58,5 +58,44 @@ def compare_forks(self, cur_fork_head, new_fork_head):
             Boolean: True if the new chain should replace the current chain.
             False if the new chain should be discarded.
         """
-        LOGGER.debug("PoetForkResolver.compare_forks()")
-        return new_fork_head.block_num > cur_fork_head.block_num
+        if new_fork_head.block_num > cur_fork_head.block_num:
+            LOGGER.info(
```

Comment:
Yes.  However, while working on this I want those messages in the log to go with the chain/publisher messages associated with them.